algorithmic_efficiency/spec.py

```diff
@@ -369,7 +351,33 @@ def init_optimizer_state(workload: Workload,
   pass
 
 
-_UpdateReturn = Tuple[OptimizerState, ParameterContainer, ModelAuxiliaryState]
+# def update_params(workload: Workload,
```

Comment:
can you delete this commented out code?

Reply:
Thank you for the comment! I deleted the debugging comment.